src/con.c

```diff
@@ -1298,6 +1298,14 @@ bool con_move_to_mark(Con *con, const char *mark) {
         return false;
     }
 
+    /* For target containers in the scratchpad, we just send the window to the scratchpad. */
+    Con *__i3_scratch = workspace_get("__i3_scratch", NULL);
+    if (con_get_workspace(target) == __i3_scratch) {
```

Comment:
I would prefer
```suggestion
    if (con_get_workspace(target) == workspace_get("__i3_scratch", NULL)) {
```
or
```suggestion
    if (con_is_internal(con_get_workspace(target)) {
```

Reply:
Updated